### DISCUSSION TITLE
Why not just use utility css?

### DISCUSSION THREAD

**Discussion by smithtn3** [chakra-ui/panda#717] [Q&A] [answered]
What are the benefits of Panda CSS?

Every utility CSS method appears to offer all of the Pands CSS advantages, so why create another CSS library?

The only benefit I see is the ability to produce hashed class names to avoid conflicting class names, but even if that is the case, why not just build a css-in-js method of utility css like https://twind.style/ ?

**Comment by paulm17** [comment, marked as answer]
The benefits are that panda css is the leading library in terms of features.  A bit old, but see:  https://github.com/andreipfeiffer/css-in-js/blob/main/README.md#overview

If you have taken a look at all the other css-in-js libraries that are available (like I have).  It would become clear why the team developed panda css.

**Reply by smithtn3** [reply]
hmmm... hope somebody can add Panda CSS to the comparison table you provided.

**Comment by tanftw** [comment]
For me, there are two main points for choosing Panda over utility CSS libraries like Tailwind:

- Compose components with `cva` and `styled` function is the easiest option.
- Custom colors via CSS variables instead of classes. So, it's simpler for me to make dark mode.
